articles/data-factory/transform-data-databricks-jar.md

```diff
@@ -59,7 +59,7 @@ definition:
 |description|Text describing what the activity does.|No|
 |type|For Databricks Jar Activity, the activity type is DatabricksSparkJar.|Yes|
 |linkedServiceName|Name of the Databricks Linked Service on which the Jar activity runs. To learn about this linked service, see [Compute linked services](compute-linked-services.md) article.|Yes|
-|mainClassName|The full name of the class containing the main method to be executed. This class must be contained in a JAR provided as a library.|Yes|
+|mainClassName|The full name of the class containing the main method to be executed. This class must be contained in a JAR provided as a library. Multiple main classes may exist in one JAR.|Yes|
```

Comment:
Consider revising "Multiple main classes may exist in one JAR" to " A JAR file can contain multiple classes. Each of the classes can contain a main method. "